app/public/css/nufiPreviewPageStyles.css

```diff
@@ -0,0 +1,397 @@
+/* NuFi PREVIEW PAGE */
```

Comment:
for the sake of consistency with names of other files around, I'd name the file `nufi-preview-page-styles.css